.env.sample

```
@@ -1,9 +1,9 @@
 ############################### DEFAULT #####################################
 # Chain ID
-CHAIN_ID=167004
+CHAIN_ID=167005
 
 # Exposed ports
-PORT_L2_EXECTION_ENGINE_HTTP=8545
+PORT_L2_EXECTION_ENGINE_HTTP=8547
 PORT_L2_EXECTION_ENGINE_WS=8546
 PORT_L2_EXECTION_ENGINE_METRICS=6060
 PORT_L2_EXECTION_ENGINE_P2P=30303
```

```suggestion
PORT_L2_EXECTION_ENGINE_HTTP=8547
PORT_L2_EXECTION_ENGINE_WS=8546
PORT_L2_EXECTION_ENGINE_METRICS=6060
PORT_L2_EXECTION_ENGINE_P2P=30303
```

we should rename these vars to `PORT_L2_EXECUTION_ENGINE_*` if possible